src/constants.ts

```diff
@@ -1,31 +1,32 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-export const deploymentFileName: string = '.deployment';
-export const deploymentFile: string = `[config]
-SCM_DO_BUILD_DURING_DEPLOYMENT=true`;
-
-export const extensionPrefix: string = 'appService';
-export enum runtimes {
-    node = 'node',
-    php = 'php',
-    dotnetcore = 'dotnetcore',
-    ruby = 'ruby'
-}
-
-export function getIgnoredFoldersForDeployment(runtime: string): string[] | undefined {
-    switch (runtime) {
-        case runtimes.node:
-            return ['node_modules{,/**}'];
-        default:
-            return undefined;
-    }
-}
-
-export enum configurationSettings {
-    zipIgnorePattern = 'zipIgnorePattern',
-    showBuildDuringDeployPrompt = 'showBuildDuringDeployPrompt',
-    showRemoteFiles = 'showRemoteFiles'
-}
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
```

Comment:
Maybe it's the line-endings that's causing the problem? You can check the status bar in VS Code:
![screen shot 2018-04-19 at 9 32 04 am](https://user-images.githubusercontent.com/11282622/39005276-a7ac08f8-43b4-11e8-8099-1f975a248ecc.png)

Looks like it should be 'CLRF' for this file (we don't have a strict LF vs CLRF rule - it should just remain constant per file after it was created). If yours doesn't say CLRF, you can click that in the status bar and it should prompt you to switch line endings:
![screen shot 2018-04-19 at 9 34 50 am](https://user-images.githubusercontent.com/11282622/39005382-ee6a119a-43b4-11e8-8e0f-3e0cfc00a212.png)

Reply:
So, it does say LF at the bottom; however, when I change it to CRLF and click save, it just switches back to LF and doesn't show any changes in the SCM...

Reply:
There's probably some VS Code setting controlling that. At least now we know what the change is. I think we've spent enough time on the whitespace stuff - fix the other comment and we should be good to merge this